scss/civicrm/campaign/_campaign.scss

```diff
@@ -0,0 +1,43 @@
+.page-civicrm-campaign {
+  .crm-content-block {
+    @include box-shadow($box-shadow);
+
+    .dataTables_wrapper,
+    .crm-form-block {
+      box-shadow: none;
+    }
+
+    .dataTables_wrapper {
+      margin-bottom: 0;
+
+      > table {
+        thead th:nth-child(2),
+        tbody td:nth-child(2) {
+          padding-left: 20px;
+        }
+      }
+    }
+
+    > #mainTabContainer {
+      border-radius: $border-radius-base;
```

Comment:
why just for this page?

Reply:
outer most element should have 2px border radius, as this page is having tab container as outer element, we give radius here

Reply:
can't we make it generic then, for every page with "having tab container as outer element"?

Reply:
No as I this pages are rare and there is not real identification that can tell us to recognize that tab container is outer element.